alvr/xtask/src/dependencies.rs

```diff
@@ -157,69 +202,65 @@ pub fn build_ffmpeg_linux(nvenc_flag: bool) {
-                &temp_download_dir
-            )
-            .unwrap();
+        let codec_header_version = "12.1.14.0";
+        let temp_download_dir = deps_path.join("dl_temp");
+        command::download_and_extract_zip(
+            &format!("https://github.com/FFmpeg/nv-codec-headers/archive/refs/tags/n{codec_header_version}.zip"),
+            &temp_download_dir
+        )
+        .unwrap();
 
-            let header_dir = download_path.join("nv-codec-headers");
-            let header_build_dir = header_dir.join("build");
-            fs::rename(
-                temp_download_dir.join(format!("nv-codec-headers-n{codec_header_version}")),
-                &header_dir,
-            )
-            .unwrap();
-            fs::remove_dir_all(temp_download_dir).unwrap();
-            {
-                let make_header_cmd =
-                    format!("make install PREFIX='{}'", header_build_dir.display());
-                let _header_push_guard = sh.push_dir(&header_dir);
-                cmd!(sh, "bash -c {make_header_cmd}").run().unwrap();
-            }
-
-            let cuda = pkg_config::Config::new().probe("cuda").unwrap();
-            let include_flags = cuda
-                .include_paths
-                .iter()
-                .map(|path| format!("-I{}", path.to_string_lossy()))
-                .reduce(|a, b| format!("{a} {b}"))
-                .expect("pkg-config cuda entry to have include-paths");
-            let link_flags = cuda
-                .link_paths
-                .iter()
-                .map(|path| format!("-L{}", path.to_string_lossy()))
-                .reduce(|a, b| format!("{a} {b}"))
-                .expect("pkg-config cuda entry to have link-paths");
-
-            let nvenc_flags = &[
-                "--enable-encoder=h264_nvenc",
-                "--enable-encoder=hevc_nvenc",
-                "--enable-nonfree",
-                "--enable-cuda-nvcc",
-                "--enable-libnpp",
-                "--nvccflags=\"-gencode arch=compute_52,code=sm_52 -O2\"",
-                &format!("--extra-cflags=\"{include_flags}\""),
-                &format!("--extra-ldflags=\"{link_flags}\""),
-            ];
-
-            let env_vars = format!(
-                "PKG_CONFIG_PATH='{}'",
-                header_build_dir.join("lib/pkgconfig").display()
-            );
-            let flags_combined = flags.join(" ");
-            let nvenc_flags_combined = nvenc_flags.join(" ");
-
-            let command = format!(
-                "{env_vars} ./configure {install_prefix} {flags_combined} {nvenc_flags_combined}"
-            );
-
-            cmd!(sh, "bash -c {command}").run().unwrap();
+        let header_dir = deps_path.join("nv-codec-headers");
+        let header_build_dir = header_dir.join("build");
+        fs::rename(
+            temp_download_dir.join(format!("nv-codec-headers-n{codec_header_version}")),
```

Comment:
we have nv headers in release pipeline, should we remove them if they are now included during build?